examples/src/main/python/pi_app.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import print_function
```

Comment:
I referred this from Spark, not sure why Spark needs to put this in the top of file.